### ISSUE TITLE
outdated command does not support --minor-only

### ISSUE THREAD

**Issue by micheh** [composer/composer#5994]
According to the change log, version 1.3.0 introduced the `--minor-only` flag to the `outdated` command: 
> Added --minor-only to the outdated command to only show updates to minor versions and ignore new major versions

However, this flag is only supported by the `show` command (see #5552).

Example:

```
$ composer show --outdated --minor-only
phpmailer/phpmailer v5.2.17 v5.2.19 PHPMailer is a full-featured email creation and transfer class for PHP
```

```
$ composer outdated --minor-only                                           
  [Symfony\Component\Console\Exception\RuntimeException]  
  The "--minor-only" option does not exist.               
                                                          
outdated [-o|--outdated] [-a|--all] [-D|--direct] [--strict] [--] [<package>]
```

**Comment by alcohol** [comment]
`outdated` is an alias for `show --outdated`, but mostly through the use of a bunch of conditional expressions. It would appear `--minor-only` is not being checked for.

**Comment by Seldaek** [comment]
Fixed by 5ecd9ace593992aa0cd7523abeafa51242c5e4fd
